locale/Makefile

```diff
@@ -0,0 +1,6 @@
+DOMAIN = hammer-cli-foreman-azure-rm
+VERSION = $(shell bundle exec ruby -C ../ -e 'require "rubygems"; spec = Gem::Specification::load("../hammer_cli_foreman_kubevirt.gemspec"); puts spec.version')
```

Comment:
do we need `hammer_cli_foreman_kubevirt.gemspec` here or `hammer_cli_foreman_azure_rm.gemspec`?

Reply:
it should be azure ;)

Reply:
Raised PR -> https://github.com/theforeman/hammer_cli_foreman_azure_rm/pull/11

Reply:
Apologies for merging with that line.
Thanks @kgaikwad for the catch!
Thanks @upadhyeammit for the quick solution in #11.